weblab/experiments/tests/test_templatetags.py

```diff
@@ -14,6 +15,17 @@ def test_url_experiment_comparison_json():
     assert exp_tags.url_experiment_comparison_json([]) == '/experiments/compare/info'
 
 
+@pytest.mark.django_db
+def test_dataset_options(logged_in_user, client):
```

Comment:
To make this a bit more robust, I'd use the `other_user` fixture as well, and create a private dataset owned by other_user linked to the same protocol, so we're certain private datasets won't be revealed by this query.